testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestHiveAndDeltaLakeRedirect.java

```diff
@@ -144,8 +144,7 @@ public void testHiveToUnpartitionedDeltaPartitionsRedirectFailure()
 
         try {
             assertQueryFailure(() -> onTrino().executeQuery(format("SELECT * FROM hive.default.\"%s$partitions\"", tableName)))
-                    .hasMessageMatching(".*Table 'hive.default.test_delta_lake_unpartitioned_table_.*\\$partitions' redirected to 'delta.default.test_delta_lake_unpartitioned_table_.*\\$partitions', " +
-                            "but the target table 'delta.default.test_delta_lake_unpartitioned_table_.*\\$partitions' does not exist");
+                    .hasMessageMatching(".*Invalid Delta Lake table name \\(unknown type 'partitions'\\): test_delta_lake_unpartitioned_table_.*\\$partitions");
```

Comment:
Does the leading `.*` contain redirection information? I feel the original message is better than the new one.

Reply:
Good catch. You are right.